test/testResourceString.js

```diff
@@ -1035,6 +1035,25 @@ module.exports.resourcestring = {
         test.done();
     },
 
+    testContextResourceStringCkeanHashKey: function(test) {
```

Comment:
minor nitpick: should be "Clean" instead of "Ckean"

Reply:
Thanks! I fixed it.